ercc/registry/registry_test.go

```diff
@@ -0,0 +1,339 @@
+package registry_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/attestation"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/registry"
+	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/registry/mocks"
+	"github.com/hyperledger-labs/fabric-private-chaincode/internal/protos"
+	"github.com/hyperledger-labs/fabric-private-chaincode/internal/utils"
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+	"github.com/hyperledger/fabric-protos-go/ledger/queryresult"
+	"github.com/hyperledger/fabric-protos-go/peer/lifecycle"
+	"github.com/hyperledger/fabric/protoutil"
+	"github.com/stretchr/testify/require"
+)
+
+//go:generate counterfeiter -o mocks/transaction.go -fake-name TransactionContext . transactionContext
+type transactionContext interface {
+	contractapi.TransactionContextInterface
+}
```

Comment:
Wouldn't this in principle be unecessary if you use the third-party variant of generation where the argument is <package-path>.<Interface>. (Not that i'm asking for change, just trying to understand ....)

Reply:
maybe. cannot answer this without trying out.